CHANGELOG.md

```
@@ -2,6 +2,14 @@
 
 All notable changes to this project will be documented in this file. For commit guidelines, please refer to [Standard Version](https://github.com/conventional-changelog/standard-version).
 
+## v2.0.0
+
+ **New Features**:
+ - New activity logs for user activity.
+  - charts and historical data
```

markdownlint-cli2 (0.22.1)

[warning] Inconsistent indentation for list items at the same level
Expected: 1; Actual: 2

(MD005, list-indent)